src/commands/github.ts

```diff
@@ -0,0 +1,270 @@
+
+						embedReply.addField("Author", generateGitHubUserLink(jsonData.author), true);
+						embedReply.addField(
+							"Stats",
+							`Total: ${jsonData.stats.total}\nAdditions: ${jsonData.stats.additions}\nDeletions: ${jsonData.stats.deletions}`,
+							true
+						);
+
+						embedReply.addField("Signed", jsonData.commit.verification.verified, true);
+						embedReply.addField("Message", jsonData.commit.message, false);
+
+						const fileLength = jsonData.files.length;
+						let longestLength = 0;
+						let longestChange = 1;
+
+						jsonData.files.forEach(file => {
+							const fileNameLength = file.filename.length;
+							if (fileNameLength > longestLength) longestLength = fileNameLength;
+							if (file.additions >= 10 && longestChange == 1) longestChange = 2;
+						});
+
+						if (fileLength > 0) {
+							let count = 0;
+							let toAdd = "```";
+
+							jsonData.files.forEach(file => {
+								if (count <= 5) {
+									count += 1;
+									const spacing = longestLength - file.filename.length;
+									const pipe = longestChange - (file.additions >= 10 ? 1 : 0);
+									toAdd = `${toAdd}\n[${file.status.slice(0, 1).toUpperCase()}]${
+										file.filename
+									}: ${" ".repeat(spacing)} +${file.additions}${" ".repeat(pipe)}| -${
+										file.deletions
+									}`;
+								}
+							});
+
+							if (fileLength - count > 0) toAdd = `${toAdd}\n...and ${fileLength - count} more.`;
+
+							embedReply.addField("Files", `${toAdd}\n\`\`\``, false);
+						}
+
+						interaction.editReply(embedReply).catch(console.error.bind(console));
+					})
+					.catch(res => handleError(interaction, res as string));
+
+				break;
+			}
+			case "issue": {
+				oauthFetch(`https://api.github.com/repos/${repo}/issues/${query}`)
+					.then(json => {
+						const modifiedEmbed = handleIssueOrPr(state.config, json as Issue, embedReply);
+
+						interaction.editReply(modifiedEmbed).catch(console.error.bind(console));
+					})
+					.catch(res => handleError(interaction, res as string));
+
+				break;
+			}
+			case "pull-request":
+			case "pullrequest":
+			case "pull request":
+			case "pr": {
+				oauthFetch(`https://api.github.com/repos/${repo}/pulls/${query}`)
+					.then(json => {
+						const jsonData = json as PullRequest;
+						const modifiedEmbed = handleIssueOrPr(state.config, jsonData, embedReply);
+
+						modifiedEmbed.addField("Merged", jsonData.merged, true);
+						if (!jsonData.merged) {
+							modifiedEmbed.addField("Mergeable", jsonData.mergeable, true);
+						} else {
+							modifiedEmbed.addField("Merged at", jsonData.merged_at, true);
+							modifiedEmbed.addField("Merged by", generateGitHubUserLink(jsonData.merged_by), true);
+						}
+
+						interaction.editReply(modifiedEmbed).catch(console.error.bind(console));
+					})
+					.catch(res => handleError(interaction, res as string));
+
+				break;
+			}
+			case "repo":
+			case "repository":
```

Comment:
Inconsistent bracing around case stmt, others have open+close brace this one doesn't

Reply:
removed all other braces as they are not needed